docs/xstate/actions/built-in-actions.mdx

```diff
@@ -47,6 +47,122 @@ send((context, event) => {
 });
 ```
 
+### Sending events to actors
+With the `send` action, events can be sent to actors via a [target expression](#send-targets):
+
+```ts twoslash {12-16}
+// @noErrors
+import { assign, createMachine, send, spawn } from 'xstate';
+
+const machine = createMachine({
+  // ...
+  states: {
+    active: {
+      entry: assign({
+        someRef: () => spawn(someMachine)
+      }),
+      on: {
+        SOME_EVENT: {
+          // use a target expression to send an event
+          // to the actor reference
+          actions: send(
```

Comment:
`sendTo` should be preferred for sending events to actors (we're deprecating `send`), but that can be a future change once we finally deprecate `send`.

Reply:
I'm happy to update it now. I've tried to do that in the latest commit; let me know what you think.

Reply:
I'll merge this; we can always edit later if something needs rewriting 😊